source/Scripts/RootConfiguration.ps1

```diff
@@ -2,29 +2,31 @@ configuration RootConfiguration
 {
     #<importStatements>
 
+    $rsopCache = Get-DatumRsopCache
```

Comment:
This is there again, I hope it's accessing a Module variable so it does not redo the work.
`Get-DatumRsopCache` is not yet in this draft PR so I can't tell